cli/flags.rs

```diff
@@ -662,7 +664,12 @@ fn fetch_subcommand<'a, 'b>() -> App<'a, 'b> {
     .arg(importmap_arg())
     .arg(config_arg())
     .arg(no_remote_arg())
-    .arg(Arg::with_name("file").takes_value(true).required(true))
+    .arg(
+      Arg::with_name("file")
+        .takes_value(true)
+        .required(true)
+        .min_values(1),
+    )
     .about("Fetch the dependencies")
     .long_about(
```

Comment:
At the bottom of this file there are some unit tests, please add one to demo the new CLI possibility.

Reply:
Done!